src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -324,6 +321,7 @@ export class DictionaryModal extends React.Component {
                       componentClass="textarea"
                       id="dictionary_description"
                       name="description"
+                      placeholder="e.g This is my organisation's dictionary"
```

Comment:
Good job. However, I think a better phrase can be used for the placeholder that tells the user the use of this field.

Reply:
Perhaps you can give me an example from the way you would like it to be.

Reply:
`Description of this dictionary`, because a dictionary can belong to a user or an organisation.